public/locales/en/explore.json

```diff
@@ -31,5 +31,35 @@
   "base": "base",
   "version": "version",
   "codec": "codec",
-  "multihash": "multihash"
+  "multihash": "multihash",
+  "tour": {
+    "projects": {
+      "title": "Featured datasets",
+      "paragraph1": "Explore the featured datasets or paste a CID to see how that data is structured and linked across protocols."
+    },
+    "explorer": {
+      "step1": {
```

Comment:
It may be useful to include more context in key names, eg `step1_ipldExplorer`. 

This small tweak makes translator's life bit easier and removes any confusion if we decide to change the order in the future.

Reply:
I followed the following structure for (almost) every page:

```
// We only have one single tour per page,
// so the steps are directly inside the tour
"tour": {
	"stepX: { ... }
	"stepY: { ... }
}
```

The odd case here is the IPLD Explorer because we have two tours, so I split them:

```
// As we have two tours for I've created another hierarchy 
"tour": {
	// Explore projects page
	"pageX": {
		"stepX: { // steps }
	},
	// Inside the explore page
	"pageY": {
		"stepY: { //steps }
	}
}
```

The reasoning for not being explicit on the steps is because they can change often, so it's easier to just have `step1`, `step2`, `step3` to not have to think about the steps title if we need to move them around or add/remove steps.

Reply:
I've looked at how Transifex handles situation when we change `step1` to something totally different than `IPLD Explorer` and it will remove existing translations, but thanks to Shared Translation Memory we have enabled there, if the same source string appears under different key, all translations will automatically be restored there.

**tl;dr** you are right, ignore my comment, it will be fine :-)